machine.go

```diff
@@ -51,24 +51,19 @@ const (
 	defaultFirecrackerInitTimeoutSeconds = 3
 )
 
-// SeccompLevelValue represents a secure computing level type.
-type SeccompLevelValue int
+// Secure computing configuration
+type SeccompConfig struct {
+	// enabling the default seccomp filters
+	Enabled bool
 
-// secure computing levels
-const (
-	// SeccompLevelDisable is the default value.
-	SeccompLevelDisable SeccompLevelValue = iota
-	// SeccompLevelBasic prohibits syscalls not whitelisted by Firecracker.
-	SeccompLevelBasic
-	// SeccompLevelAdvanced adds further checks on some of the parameters of the
-	// allowed syscalls.
-	SeccompLevelAdvanced
-)
-
-func (level SeccompLevelValue) String() string {
-	return strconv.Itoa(int(level))
+	// Supplying user-provided, custom filters
+	CustomFilter string
```

Comment:
Is it a filepath to something?

Reply:
this is a file path

Reply:
How about calling this field `Filter`, since Firecracker side is just `--seccomp-filter`?

Reply:
updated.